src/aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -921,6 +923,23 @@ def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None)
     parsed_data['structure'] = structure_data
     parsed_data['trajectory'] = trajectory_data
 
+    # Double check to detect error messages if CRASH is found
+    if crash_file is not None:
+        lines = crash_file.split('\n')
+
+        for line_number, line in enumerate(lines):
+            # Compare the line to the known set of error and warning messages and add them to the log container
+            detect_important_message(logs, line)
+
+        if len(logs.error) or len(logs.warning) > 0:
+            parsed_data['bands'] = bands_data
+            parsed_data['structure'] = structure_data
+            parsed_data['trajectory'] = trajectory_data
```

Comment:
These are already set in lines 922 - 924 and since you don't change them, no need to set again., right?

Reply:
Correct, thanks!